Wikipedia/Code/WMFImageController.swift

```diff
@@ -411,54 +390,39 @@ extension WMFImageController {
      - returns: `AnyPromise` which resolves to `WMFImageDownload`.
      */
     @objc public func fetchImageWithURL(url: NSURL?) -> AnyPromise {
-        return AnyPromise(bound: fetchImageWithURL(url))
-    }
-    
-    /**
-     Objective-C-compatible variant of fetchImageWithURL(url:options:) returning an `AnyPromise`.
-     
-     - returns: `AnyPromise` which resolves to `WMFImageDownload`.
-     */
-    @objc public func fetchImageWithURL(url: NSURL?, options: SDWebImageOptions) -> AnyPromise {
-        return AnyPromise(bound: fetchImageWithURL(url, options: options))
+        return AnyPromise(bound: Promise<WMFImageDownload> { fulfill, reject in
```

Comment:
Are we still using Promiskit here?

Also I think we no longer need to have an objective-c compatible version - the normal `fetchImageWithURL` should work now because it was only incompatible because promisekit.